components/button/package.json

```diff
@@ -19,7 +19,7 @@
     "@spectrum-css/vars": "^8.0.0"
   },
   "devDependencies": {
-    "@spectrum-css/component-builder": "^3.2.0",
+    "@spectrum-css/component-builder-simple": "^1.0.0-beta.0",
     "@spectrum-css/icon": "^3.0.23",
     "@spectrum-css/vars": "^8.0.0",
```

Comment:
We'll remove this line, too. Instead, we need to require the tokens package.

`"@spectrum-css/tokens": "^1.0.2"`

Please also make the tokens package a `peerDependency`.